tools/bismark/bismark_deduplicate/bismark_deduplicate_wrapper.xml

```diff
@@ -0,0 +1,70 @@
+<tool id="bismark_deduplicate" name="Bismark Deduplicate" version="0.10.2">
+
+    <!-- Wrapper compatible with Bismark version 0.10 -->
+    <description>Deduplicates reads mapped by Bismark</description>
+    <!--<version_command>bismark version</version_command>-->
+
+    <requirements>
+        <requirement type="package" version="0.1.19">samtools</requirement>
+        <requirement type="package" version="2.1.0">bowtie2</requirement>
+    </requirements>
+
+    <stdio>
+        <exit_code range="1:" />
+        <exit_code range=":-1" />
+        <regex match="Error:" />
+        <regex match="Exception:" />
+    </stdio>
+
+    <command interpreter="python">
+<![CDATA[
+        bismark_deduplicate_wrapper.py
+
+        --tool_dir $__tool_directory__
```

Comment:
All path should be quoted like `$__tool_directory__`. How knows maybe someone serves Galaxy in `/my great path/foo/`